src/frequenz/sdk/power/_distribution_algorithm.py

```diff
@@ -396,24 +396,26 @@ def _greedy_distribute_remaining_power(
         Returns:
             Return the power for each inverter in given distribution.
         """
-        if remaining_power == 0:
+        if isclose(remaining_power, 0.0, abs_tol=1e-9):
```

Comment:
I would add a comment saying why you are choosing that specific absolute tolerance (and not the default relative tolerance).

Reply:
Added a brief comment and reference for the function you suggested in https://github.com/frequenz-floss/frequenz-sdk-python/pull/337#discussion_r1176616976